src/adapters/IDB.js

```diff
@@ -81,6 +87,62 @@ export default class IDB extends BaseAdapter {
   }
 
   /**
+   * Batch operations.
+   *
+   * @param  {Function} fn The operation function, which must return a Promise.
+   * @return {Promise}
```

Comment:
I definitely feel we could v-carry data over the returned value, though I don't know where to put it… `{operations, errors, result}`?